.gitignore

```diff
@@ -1,152 +1,5 @@
-# Editor files
+**/tags
 *.swp
-
-# Build directories
-.deps/
-.libs/
-
-# Build files
-*.dll
-*.exe
-*.exp
-*.la
-*.lib
-*.lo
-*.o
-*.obj
-*.pdb
-*.pyc
-.dirstamp
-.sconf_temp
-.sconsign.dblite
-/Makefile.am
-/aclocal.m4
-/autom4te.cache/
-/build_posix/aclocal/libtool.m4
-/build_posix/aclocal/ltoptions.m4
-/build_posix/aclocal/ltsugar.m4
-/build_posix/aclocal/ltversion.m4
-/build_posix/aclocal/lt~obsolete.m4
-/build_posix/config.hin
-/build_posix/gnu-support/
-/config.log
-/config.status
-/configure
-/configure.ac
-/libtool
-/stamp-h1
-/wiredtiger.h
-/wiredtiger.pc
-/wiredtiger_config.h
-/wiredtiger_ext.h
-/wt
-Makefile
-Makefile.in
-build_posix/config.log
-build_posix/config.status
-build_posix/libtool
-build_posix/stamp-h1
-build_posix/wiredtiger.h
-build_posix/wiredtiger.pc
-build_posix/wiredtiger_config.h
-build_posix/wiredtiger_ext.h
-build_posix/wt
-tags
-
-# Documentation
-/dist/plantuml.jar
-/docs/
-/src/docs/changelog.md
-/src/docs/doxygen.log
-
-# Tests
-WT_HOME/
-WT_TEST/
-
-# Bench/workgen
-/bench/workgen/_workgen.so
-/bench/workgen/workgen/workgen.py
-/bench/workgen/workgen_wrap.cxx
-
-# Python
-/lang/python/_wiredtiger.so
-/lang/python/wiredtiger.py
-/lang/python/wiredtiger/__init__.py
-/lang/python/wiredtiger/_wiredtiger.pyd
-/lang/python/wiredtiger_wrap.c
-_wiredtiger.pyd
-
-# Binaries
-**/bench/wtperf/wtperf
```

Comment:
Would these `**` paths still be useful to match build binaries in any out-of-source build directory, given the build directory doesn't necessarily need to be called `build`?
All good if its cleaner to remove since any additional build directories pop up as untracked files.

Reply:
I couldn't think of any way to ignore random build directories (absent listing every possible file in a build directory and ignoring them explicitly, with `**` as you suggested, and that seems like a real pain to deal with).